examples/run.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) 2020 Fredy Ramirez <https://formateli.com>
+# For license see LICENSE
+"""
+run.py
+Show all examples located in this example folder.
+Main window show a button for each example.
+"""
+import os
+import os
+import subprocess
+import tkinter as tk
+from tkinter import ttk
+
+_DIR = os.path.dirname(os.path.realpath(__file__))
+_DIR = os.path.normpath(os.path.join(_DIR, '..'))
+
+EXAMPLE_ENV = os.environ.copy()
+if not 'PYTHONPATH' in EXAMPLE_ENV:
+    EXAMPLE_ENV["PYTHONPATH"] = _DIR
+else:
+    EXAMPLE_ENV["PYTHONPATH"] = _DIR + os.pathsep + EXAMPLE_ENV["PYTHONPATH"]
+
+
+class _SampleButton:
+    def __init__(self, root, text, col, row):
+        self.root = root
+        self.btn = ttk.Button(root, text=text, command=self.run_example)
+        self.btn.grid(row=row, column=col, sticky="nsew") 
+
+    def run_example(self, event=None):
+        subprocess.run(
```

Comment:
This really may just be on a single line. So like:
```python
subprocess.run(['python', 'example_' + self.btn['text'] + '.py'], env=EXAMPLE_ENV)
```
Also, what do you think about changing this to a `subprocess.Popen`? Then the runner program won't be blocked while running examples. It's a bit counter-intuitive when the program is blocked, a button for another example is clicked and this click only gets processed when the other example is closed, immediately opening the new example.

Reply:
Sorry, I am used to 80 characters per line. Corrected.
You are right, it is much better using subprocess.Popen. Done.